vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -500,6 +525,27 @@ XamlView ScrollViewManager::CreateViewCore(int64_t /*tag*/, const winrt::Microso
   return scrollViewer;
 }
 
+void ScrollViewManager::SetLayoutProps(
+    ShadowNodeBase &nodeToUpdate,
+    const XamlView &viewToUpdate,
+    float left,
+    float top,
+    float width,
+    float height) {
+  // ScrollViewer selects an anchor during the Arrange phase of layout.
+  // If you do not call InvalidateArrange whenever a new child is added
+  // to the ScrollViewer content, the anchor behavior does not work.
+  //
+  // While this call fires too frequently resulting in unnecessary
+  // calls to invalidate arrange, it is the only sure-fire way to call
+  // InvalidateArrange any time any descendent layout changes.
+  if (static_cast<ScrollViewShadowNode &>(nodeToUpdate).IsInverted()) {
```

Comment:
Consider conditioning on whether we are anchoring instead of inverted, since there may be future cases with anchoring without inversion